src/main/java/net/sf/jabref/gui/autocompleter/AutoCompleteState.java

```diff
@@ -0,0 +1,65 @@
+package net.sf.jabref.gui.autocompleter;
+
+import java.util.List;
+
+public class AutoCompleteState {
+    private List<String> suggestions;
+    private String currentWord;
+    private int currentCompletion;
+    private int realCaretPosition;
+
+    public AutoCompleteState(String currentWord, List<String> suggestions, int caretPosition) {
+        this.currentWord = currentWord;
+        this.suggestions = suggestions;
```

Comment:
Use Objects.requireNonNull for both values like this.field = requireNonNull(field);